EVENT.md

```diff
@@ -562,6 +563,22 @@ fully understood during roll-out and help flush out any bugs (either in browser
 or caller code), and more easily compare the performance to cookie-based
 alternatives.
 
+To ensure that this data (which could enable cross-site tracking) is only
+available in a transitional phase while third-party cookies are available and
+are already capable of user tracking, the browser will check (at both source
+and trigger registration) for the presence of a special `SameSite=None` cookie
```

Comment:
```suggestion
and trigger registration) for the presence of a special cookie
```
No need to repeat this since it's in the example code immediately below.